pipeline/frontend/yaml/container.go

```diff
@@ -42,6 +42,7 @@ type (
 		ExtraHosts    []string               `yaml:"extra_hosts,omitempty"`
 		Group         string                 `yaml:"group,omitempty"`
 		Image         string                 `yaml:"image,omitempty"`
+		IgnoreFailure bool                   `yaml:"ignore_failure,omitempty"`
```

Comment:
Or should we use `failure: ignore` like drone?

Reply:
depends if there could be other options beside ignore/fail?

Reply:
:point_up: 

you normaly have three options, fail hard (cancle running steps), fail (let already running steps be finished **we do this atm**), ignore

fail hard will only have an efect if failing step is in a group (parallel exec)

Reply:
Maybe we can have:

failure:
* fail (default)
* cancel
* ignore

Reply:
The different ways to handle a failed step make sense to me, I'll have a go at implementing it :)